File: C/Assignment/For/prg7.c
#include<stdio.h>

void main() {

    int value, range;

    printf("Enter the value\n");
    scanf("%d", &value);

    printf("Enter upper most range\n");
    scanf("%d", &range);

    for(int i = value; i <= range; i += value)
        printf("%d\t", i);

    printf("\n");
}

File: Python/Try/prg7.py
arr = [[1, 2]] * 3
print(arr)
print(id(arr[0]))
print(id(arr[1]))
print(id(arr[2]))
arr[0][0] = 10
print(arr)

print('\n\n')

arr = [[j for j in range(1, 3)] for i in range(3)]
print(arr)
print(id(arr[0]))
print(id(arr[1]))
print(id(arr[2]))
arr[0][0] = 10
print(arr)
